### DISCUSSION TITLE
Can't install Globaleaks

### DISCUSSION THREAD

**Discussion by robertodc** [globaleaks/globaleaks-whistleblowing-software#3767] [Q&A]
As I am new of Linux, and I need to install Globaleaks, how to do, as I get errors?
I got the install-globaleaks.sh file in whistleblowing fold
but I cant execute the other 2 commands:
chmod +x install-globaleaks.sh I got the error: operation not permitted
./install-globaleaks.sh - in the root I got the error: not such file or directory
Thank you.

**Comment by evilaliv3** [comment]
Hello.

The reason of the first error was that you were not operating as root.
The reason of the second error was that you were operating as root but probably in a directory different from the one where you downloaded the install script.

**Comment by robertodc** [comment]
Thank you, but after my command: sudo ./install-globaleaks.sh
I get this, and I don't know what to do:

Preparing to unpack .../tzdata_2023c-0ubuntu0.22.04.2_all.deb ...
Unpacking tzdata (2023c-0ubuntu0.22.04.2) over (2023c-0ubuntu0.22.04.1) ...
Setting up tzdata (2023c-0ubuntu0.22.04.2) ...

Current default time zone: 'Europe/Rome'
Local time is now:      Fri Nov 10 15:31:09 CET 2023.
Universal Time is now:  Fri Nov 10 14:31:09 UTC 2023.
Run 'dpkg-reconfigure tzdata' if you wish to change it.


Current default time zone: 'Europe/Rome'
Local time is now:      Fri Nov 10 15:31:10 CET 2023.
Universal Time is now:  Fri Nov 10 14:31:10 UTC 2023.

Running: "apt-get -y install curl gnupg net-tools software-properties-common"... SUCCESS
Running: "is_tcp_sock_free_check 0.0.0.0:80"... SUCCESS
Running: "is_tcp_sock_free_check 0.0.0.0:443"... SUCCESS
Running: "is_tcp_sock_free_check 127.0.0.1:8082"... SUCCESS
Running: "is_tcp_sock_free_check 127.0.0.1:8083"... SUCCESS
 + required TCP sockets open
Adding GlobaLeaks PGP key to trusted APT keys
Updating GlobaLeaks apt source.list in /etc/apt/sources.list.d/globaleaks.list ...
Running: "apt-get update -y"... SUCCESS
Running: "apt-get install globaleaks -y"... SUCCESS
Ouch! The installation is complete but GlobaLeaks failed to start.
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State
tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN
cat: /var/globaleaks/log/globaleaks.log: No such file or directory
roby2@Roberto-PC:/mnt/c/whistleblowing$

Can u help me, please?

**Reply by robertodc** [reply]
I could install Globaleaks using docker.
But 1 question, please: shall I always keep docker running, even if when I finish to customize the program and publish the link in my web page? Or the program on the web will always work even if I close docker?
Thank you.
